dataprofiler/data_readers/csv_data.py

```diff
@@ -716,7 +733,12 @@ def is_match(cls, file_path, options=None):
         # Assume not a CSV
         return False
 
-    def reload(self, input_file_path=None, data=None, options=None):
+    def reload(
+        self,
+        input_file_path: Optional[str] = None,
+        data: Optional[pd.DataFrame] = None,
```

Comment:
`:type data: multiple types` is the docstring for `data`... but static typing is only saying `None` or `pd.DataFrame`. I think at least one or the other should be update to make sure the code and docstring match